one.js

```diff
@@ -0,0 +1,32 @@
+var gender = prompt("are you a boy or a girl?");
+
+if(gender === "boy" || gender == "girl"){
+    alert("Let's play a game");
+  }
+else{
+  alert("Start Over");
+ document.write("Entered a failed value, try again");
```

Comment:
in general, would shy away from `writing` anything directly to the document. Instead, would recommend using jQuery or Vanilla JS to `append` an element to the document.